File: Python Word Examples/MergeApiDemo/mergeDocument.py
import spirecloudword
from spirecloudword import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey, baseUrl)
api = spirecloudword.api.merge_api.MergeApi(configuration)

file_name = "Template.docx"
merging_file_list = []
merging_file_list.append(MergingFile('input/Table.docx'))
extra = {'dest_file_path': 'output/MergeDocument.docx',
         'folder': 'input'}
api.merge_document(file_name, merging_file_list, **extra)

File: JavaScript Word Examples/MergeApiDemo/mergeDocument.js
(function (){
    var appId = "your id";
    var appKey = "your key"; 
    var baseUrl = "https://api.e-iceblue.cn";

    var Spirecloudword= require('../../src/index');
    var apiClient = new Spirecloudword.ApiClient(appId, appKey,baseUrl);
    var instance = new Spirecloudword.MergeApi(apiClient);

    var name = "Template.docx";
    var files = [
        new Spirecloudword.MergingFile(), new Spirecloudword.MergingFile(), new Spirecloudword.MergingFile()
    ];
    files[0].path = "input/Merge1.docx";
    files[1].path = "input/Merge2.doc";
    files[2].path = "input/Merge3.docx";
    var opt = { folder: 'input'};
    var destFilePath = 'output/MergeDocument.docx';
    instance.mergeDocument(name, files, destFilePath, opt,
        function (error, data, response) {
            if (error) throw error;
        });
})();